File: 백준/Bronze/2581. 소수/소수.py
m=int(input())
n=int(input())
a=[]
sum=0
for i in range(m,n+1):
    count=0
    for k in range(1,i+1):
        if i % k == 0 :
            count+=1
    if count == 2:
        a.append(i)
        sum+=i
if sum == 0:
    print(-1)
else:
    print(sum)
    print(min(a))

File: 백준/Bronze/2581. 소수/소수.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int m = Integer.parseInt(br.readLine());
        int n = Integer.parseInt(br.readLine());

        int sum = 0;
        int min = Integer.MAX_VALUE;
        boolean exist = false;

        for(int i = m ; i <= n ; i++){
            boolean isPrime = true;
            if(i == 1){
                continue;
            }
            for(int j = 2; j * j <= i; j++){
                if(i % j == 0){
                    isPrime = false;
                    break;
                }
            }
            if(isPrime){
                exist = true;
                sum += i;
                if(i < min){
                    min = i;
                }
            }
        }

        if(exist){
            System.out.println(sum);
            System.out.print(min);
        }
        else{
           System.out.print(-1);
        }
    }
}